draft-ietf-mls-protocol.md

```diff
@@ -769,23 +771,9 @@ The ciphersuite must also specify an algorithm `Derive-Key-Pair`
 that maps octet strings with the same length as the output of the
 hash function to key pairs for the asymmetric encryption scheme.
 
-Public keys used in the protocol are opaque values
-in a format defined by the ciphersuite, using the following types:
-
-~~~~~
-opaque HPKEPublicKey<1..2^16-1>;
```

Comment:
We still need this definition. Maybe put it under the `CipherSuite` enum, as with `SignaturePublicKey` and the `SignatureScheme` enum.

Reply:
Also, some descriptive text like you added for `SignaturePublicKey` would be helpful.